user-service/src/main/java/xyz/bookself/users/repository/UserRepository.java

```diff
@@ -7,7 +7,6 @@
 import java.util.Collection;
 
 public interface UserRepository extends JpaRepository<User, Integer> {
-
-    @Query(nativeQuery = true, value = "SELECT * FROM users LIMIT ?1")
-    Collection<User> findUser(int limit);
-}
\ No newline at end of file
+    @Query(nativeQuery = true, value = "SELECT * FROM users WHERE email = ?1")
+    User findUserByEmail(String email);
```

Comment:
Are we having the user log in via username and/or email? If both might make sense to be able to pass in either the username or email and check for an account that has that string in either the username or email.

Reply:
I thought this was for checking if the user already had an account or not. But I can make another DTO for login.

Reply:
Oh that makes sense, I misread what this was for. Please ignore!